src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -128,14 +135,13 @@ def convert_prompt_template(template, inputs, api):
 
     # convert list type into ChatInputList type
     converted_kwargs = convert_to_chat_list(inputs)
```

Comment:
Why do we need to convert the list type for all the requests? Originally, as far as I know, the list type is converted to ChatInputList only for vison tool, not sure if we need to convert to chat input list for all of the types. + @melionel to review